server/utils/middlewareFunction.ts

```diff
@@ -0,0 +1,14 @@
+import jwt from "jsonwebtoken";

```

Comment:
file name is generic

Reply:
Okay. I will change it to something more specific.